scss/elements/_typography.scss

```diff
@@ -79,3 +79,9 @@ hr {
   border-top: none;
   margin: u(1rem 0);
 }
+
+blockquote {
+  border-left: 10px solid $gray-lightest;
+  padding-left: 30px;
+  font-size: u(1.6rem);
```

Comment:
This is actually something I just don't know, but maybe you can teach me :) 
If this is inheriting the `<p>` class styles, do we need to set the font size? Or should it have it automatically?